crates/paralegal-policy/src/flows_to.rs

```diff
@@ -47,54 +62,100 @@ impl CtrlFlowsTo {
             }
         }
 
+        fn iterate(
+            sources: &Arc<IndexedDomain<DataSource>>,
+            flows_to: &mut IndexMatrix<DataSourceIndex, CallSiteOrDataSink>,
+            sink_to_source: &IndexMatrix<CallSiteOrDataSinkIndex, DataSource>,
+        ) {
+            // Compute the transitive closure to a fixpoint.
+            loop {
```

Comment:
This might be easier to read if you make this

```rust
let mut changed = true;
while changed {
    changed = false;
    ...
}
```

That way you can avoid the `break`

Reply:
Done